app/client/components/user/singleUserManagement.jsx

```diff
@@ -145,6 +149,18 @@ class SingleUserManagement extends React.Component {
     ) && this.props.resetPin({ user_id: this.props.user.id });
   }
 
+  deleteUser() {
+    const user_id = this.props.userId.toString();
+    let del = window.prompt(
+      `Are you sure you wish to delete user "${this.state.first_name} ${this.state.last_name}"? This action cannot be undone. Type DELETE to confirm`
+    );
+    if (del === "DELETE") {
```

Comment:
Asking a user to actively type `DELETE` felt like the best way to confirm action 100%